src/main/java/org/folio/qm/service/ValidationService.java

```diff
@@ -16,5 +16,5 @@ public interface ValidationService {
 
   void validateIdsMatch(QuickMarcEdit quickMarc, UUID externalId);
 
-  void validateMarcRecord(BaseMarcRecord marcRecord);
+  void validateMarcRecord(BaseMarcRecord marcRecord, boolean is001RequiredField);
```

Comment:
checking for the 001 field with additional ```is001RequiredField ``` boolean param works well here, but it does not scale well if in future there might be more fields such as 001.

I think we could declare list of some structures, like ```ValidationIssueWithRuleCode``` you have added here, that needs to be skipped during the validation and that list then could be utilized in the implementation of create record and not in the implementation of update record. But, this idea could be left for implementing later or next time :-)

Reply:
I think it's a good idea to provide more flexible way to provide exceptions of what validation errors should be skipped.